partiql-parser/src/main/antlr/PartiQLParser.g4

```diff
@@ -541,9 +542,9 @@ exprSelect
         where=whereClauseSelect?
         group=groupClause?
         having=havingClause?
-        order=orderByClause?
-        limit=limitClause?
-        offset=offsetByClause? # SfwQuery
+        order=orderByClause??
+        limit=limitClause??
+        offset=offsetByClause?? # SfwQuery
```

Comment:
> Since the AST nodes don't allow for this, this may result in ambiguous results.

Could you clarify on this? Are you referring to if a set op query is defined without parens and it includes a `LIMIT`, `OFFSET`, or `ORDER BY`? For example,

```
SELECT * FROM a UNION SELECT * FROM b LIMIT 5
```

In this case, I believe the parsing behavior we want is to apply the `LIMIT` to the `UNION` set operator. This behavior follows what PostgreSQL's parsing does - https://www.postgresql.org/docs/current/queries-union.html.